server/config/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  // Secret for JWT signing and encryption
+  secret: process.env.SECRET || 'secret',
+  // Port for server
+  port: process.env.PORT || 3000
```

Comment:
Instead of comments, let's use readable names instead such as:
```js
{
  jwtSigningKey: process.env.JWT_SIGNING_KEY || 'secret',
  serverPort: process.env.PORT || 3000
}
```
As a general rule, soon as there are comments, it means the code is not as clear as it could be :).